README.md

```diff
@@ -0,0 +1,2 @@
+# xlineapi
```

Comment:
Plz keep consistent with the repo name.

Reply:
Already modified, does the repo curp-proto's readme file need to be modified according to this rule as well?